### PR TITLE
Fixes #8915: Syntax errors in rudder commands

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8915

### CODE REVIEW

File: share/commands/agent-disable [hunk 1]
```diff
@@ -32,6 +32,6 @@ done
 touch /opt/rudder/etc/disable-agent
 
 [ "$QUIET" = false ] && printf "${GREEN}ok${NORMAL}: Rudder agent is now disabled.\n"
-if [ "${STOP}" = "y" ]
+if [ "${STOP}" = "y" ]; then
   "${BASEDIR}/agent-stop" ${OPTIONS}
```
Comment (jooooooon) [inline share/commands/agent-disable]: This shoudl be "ruder agent stop"

Comment (amousset) [inline share/commands/agent-disable]: Indeed, but I think it should go into another PR.

Comment (amousset) [inline share/commands/agent-disable]: Opened http://www.rudder-project.org/redmine/issues/8925, which version should we target?

Comment (jooooooon) [inline share/commands/agent-disable]: OK. All versions - this could bite us one day.
